modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form/FormView.es.js

```diff
@@ -46,14 +47,16 @@ import {paginationReducer} from './reducers/index.es';
  * the submit of the Form in the User View, removes the default behavior
  * of the submit and uses Liferay.Util.submitForm.
  */
+
+const DDM_FORM_PORTLET_NAMESPACE =
```

Comment:
@mlan0, can you please move this variable declaration to be on top of the comment above?

This comment is associated to the method `submitForm`, previously named `handleFormSubmitted`. :wink:

Reply:
Done!